util.js

```diff
@@ -53,30 +53,41 @@ exports.readEnv = (forServer) => {
 /**
  * Find the script tags to include
  */
-exports.findScripts = (isDist) => {
-  let names = ['inline', 'polyfills', 'vendor', 'main'];
+const findScripts = (isDist) => {
+  let names = ['polyfills', 'main', 'runtime'];
```

Comment:
This isn't the order I'm seeing in the `ng serve [--dist]` index file.  Seems like this should be

```
let names = ['runtime', 'polyfills', 'vendor', 'main'];
```

Since the ng5 -> ng6 basically renamed `inline -> runtime`, and then dropped the `.bundle` from all the filenames.  Or am I missing something?

Reply:
Or I'm slightly wrong about that, as I don't see vendor in the prod build anymore.  Or in my local ng5 prod build, for that matter.  What the what?

Reply:
Yeah `vendor` is required for the dev server, but doesn't exist for prod.

Reply:
It certainly doesn't hurt anything to preserve order. Will update.